ee/be/modules/350-node-local-dns/templates/configmap.yaml

```
@@ -16,14 +16,20 @@ data:
         success 39936
         denial 9984
         prefetch 10 1m 25%
-        serve_stale 1h verify
+        serve_stale 1h immediate
+      }
+      dynamicforward {
+        namespace kube-system
+        service_name d8-kube-dns
+        port_name dns
+        expire 10s
+        health_check 100ms
+        prefer_udp true
```

```suggestion
        prefer_udp
```
If possible, the `dynamicforward` plugin should maintain the same format for specifying options as the `forward` plugin.

---

Our plugin has its own key value format, does this look bad?